app/views/gobierto_participation/layouts/_navigation.sub.html.erb

```diff
@@ -1,22 +1,8 @@
 <% if try(:current_process).nil? %>
-  <div class="sub-nav-item">
-    <%= link_to t('gobierto_participation.layouts.menu_subsections.about'), '#' %>
-  </div>
-  <div class="sub-nav-item<%= class_if(' active', controller_name == 'issues') %>">
-    <%= link_to t('gobierto_participation.layouts.menu_subsections.issues'), gobierto_participation_issues_path, data: { turbolinks: false } %>
-  </div>
-  <div class="sub-nav-item<%= class_if(' active', controller_name == 'scopes') %>">
-    <%= link_to t('gobierto_participation.layouts.menu_subsections.scopes'), gobierto_participation_scopes_path, data: { turbolinks: false } %>
-  </div>
-  <div class="sub-nav-item<%= class_if(' active', controller_name == 'processes') %>">
-    <%= link_to t('gobierto_participation.layouts.menu_subsections.processes'), gobierto_participation_processes_path, data: { turbolinks: false } %>
-  </div>
-  <div class="sub-nav-item">
-    <%= link_to t('gobierto_participation.layouts.menu_subsections.questions'), '#' %>
-  </div>
-  <div class="sub-nav-item">
-    <%= link_to t('gobierto_participation.layouts.menu_subsections.ideas'), '#' %>
-  </div>
+  <% liquid_path = "gobierto_participation/layouts/navigation_process.liquid" %>
+  <% liquid_str = GobiertoCore::SiteTemplate.liquid_str(current_site, liquid_path) %>
+
+  <%= raw render_liquid(liquid_str) %>
```

Comment:
I'd refactor it a bit, too many things to remember when trying to render a template.

I'd like the syntax to be something like: `<%= raw render_liquid("gobierto_participation/layouts/navigation_process") %>

- remove the calls to liquid_str from the view, and move them to the helper
- remove the extension `.liquid` of the template